stories/slider/slider.stories.js

```diff
@@ -0,0 +1,146 @@
+export default {
+    title: 'Components/Slider',
+    parameters: {
+        description: `A slider is a control that enables the user to adjust single values within a specified numerical range. A range slider is a user interface control that enables the user to select a value range within a predefined numerical
+interval.`,
+        components: ['slider']
+    },
+    tags: ['f3', 'theme']
+};
+
+/**
+ * Basic slider
+ */
+
+export const basic = () => `
+<div class="slider-container">
+    <div class="fd-slider" tabindex="0" role="slider" aria-label="slider" aria-valuemin="1" aria-valuemax="100" aria-valuenow="50">
+        <div class="fd-slider__inner">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 50%;"></div>
+                <div class="fd-slider__handle" style="left: 50%;"></div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+/**
+ * Range slider
+ */
+
+export const range = () => `
+<div class="slider-container">
+    <div class="fd-slider" tabindex="0" role="slider" aria-label="range slider" aria-valuemin="1" aria-valuemax="100" aria-valuenow="40">
+        <div class="fd-slider__inner">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 40%;left: 40%"></div>
+                <div class="fd-slider__handle" style="left: 40%;"></div>
+                <div class="fd-slider__handle" style="left: 80%;"></div>
+            </div>
```

Comment:
Similar to the single-handle examples, the handles themselves should be the ones with role `slider` with the associated ARIA attributes as they would be treated as two separate tabbable elements. [Example](https://www.w3.org/TR/wai-aria-practices/examples/slider/multithumb-slider.html)

```suggestion
    <div class="fd-slider">
        <div class="fd-slider__inner">
            <div class="fd-slider__track">
                <div class="fd-slider__track-range" style="width: 40%;left: 40%"></div>
                <div class="fd-slider__handle" tabindex="0" role="slider" aria-label="min slider" aria-valuemin="1" aria-valuemax="80" aria-valuenow="40" style="left: 40%;"></div>
                <div class="fd-slider__handle" tabindex="0" role="slider" aria-label="max slider" aria-valuemin="40" aria-valuemax="100" aria-valuenow="80" style="left: 80%;"></div>
            </div>
```